test/cargo_build_script/BUILD.bazel

```
@@ -66,3 +66,40 @@ rust_test(
     edition = "2018",
     deps = [":empty_build_script"],
 )
+
+###############################################################################
+# Test that the build script can access files in the exec root.
+#
+# All assertions are done in the build script. If it succeeds in execution, the
+# test passes.
+###############################################################################
+
+write_file(
+    name = "root_file",
+    out = "root_file.txt",
```

```suggestion
    name = "cargo_manifest_dir_file",
    out = "cargo_manifest_dir_file.txt",
```

My first read of this in the build script assumed "root" meant "root of repo"